pkg/asset/installconfig/clusterid.go

```diff
@@ -0,0 +1,25 @@
+package installconfig
+
+import (
+	"github.com/pborman/uuid"
+
+	"github.com/openshift/installer/pkg/asset"
+)
+
+type clusterID struct{}
+
+var _ asset.Asset = (*clusterID)(nil)
+
+// Dependencies returns no dependencies.
+func (a *clusterID) Dependencies() []asset.Asset {
+	return []asset.Asset{}
+}
+
+// Generate generates a new UUID
+func (a *clusterID) Generate(map[asset.Asset]*asset.State) (*asset.State, error) {
+	return &asset.State{
+		Contents: []asset.Content{
+			{Data: []byte(uuid.NewUUID().String())},
```

Comment:
Note that `uuid.NewUUID` uses a time-based seed to generate the random ID. You can use the apimachinery util instead - https://github.com/kubernetes/apimachinery/blob/017bf4f8f5884203a401400e511f4c8fc2eded21/pkg/util/uuid/uuid.go#L30

Reply:
> Note that `uuid.NewUUID` uses a time-based seed to generate the random ID.

I think we should just switch to asking for a [version 4 UUID][1].  It looks like that's [`[]byte(uuid.New())`][2] ([`NewUUID`][3] is asking for a version 1 UUID).

[1]: https://tools.ietf.org/html/rfc4122#section-4.4
[2]: https://godoc.org/github.com/pborman/uuid#New
[3]: https://godoc.org/github.com/pborman/uuid#NewUUID

Reply:
Good point, I can address this when rebasing my pr #145